modules/effects/src/shader-modules/blur-filters/tiltshift.js

```diff
@@ -36,14 +36,14 @@ vec4 tiltShift_sampleColor(sampler2D texture, vec2 texSize, vec2 texCoord) {
   /* randomize the lookup values to hide the fixed number of samples */
   float offset = random(vec3(12.9898, 78.233, 151.7182), 0.0);
 
-  vec2 normal = normalize(vec2(start.y - end.y, end.x - start.x));
+  vec2 normal = normalize(vec2((start.y - end.y) * texSize.y, (end.x - start.x) * texSize.x));
```

Comment:
Nit: it's more efficient to transform `start` and `end` once than multiplying `texSize` on every usage.